dipy/viz/ui.py

```diff
@@ -693,3 +700,523 @@ def re_align(self, window_size_change):
+            The number of characters in a single line of text.
+        height : int
+            The number of lines in the textbox.
+        window_left: int
+            Left limit of visible text in the textbox.
+        window_right: int
+            Right limit of visible text in the textbox.
+        caret_pos: int
+            Position of the caret in the text.
+        init: bool
+            Flag which says whether the textbox has just been initialized.
+
+    """
+    def __init__(self, width, height, text="Enter Text", position=(100, 10),
+                 color=(0, 0, 0), font_size=18, font_family='Arial',
+                 justification='left', bold=False,
+                 italic=False, shadow=False):
+        """
+        Parameters
+        ----------
+        width : int
+            The number of characters in a single line of text.
+        height : int
+            The number of lines in the textbox.
+        text : string
+            Initial text while placing the element.
+        position : (float, float)
+        color : (float, float, float)
+            Values must be between 0-1.
+        font_size : int
+        font_family : string
+            Currently only supports Ariel.
+        justification : string
+            left, right or center.
+        bold : bool
+        italic : bool
+        shadow : bool
+
+        """
+        self.text = text
+        self.actor = self.build_actor(self.text, position, color, font_size,
+                                      font_family, justification, bold, italic, shadow)
+        self.width = width
+        self.height = height
+        self.window_left = 0
+        self.window_right = 0
+        self.caret_pos = 0
+        self.init = True
+        super(TextBox2D, self).__init__()
+        self.on_left_mouse_button_pressed = self.left_button_press
+        self.on_key_press = self.key_press
+
+    def build_actor(self, text, position, color, font_size,
+                    font_family, justification, bold, italic, shadow):
+
+        """ Builds a text actor.
+
+        Parameters
+        ----------
+        text : string
+            The initial text while building the actor.
+        position : (float, float)
+        color : (float, float, float)
+            Values must be between 0-1.
+        font_size : int
+        font_family : string
+            Currently only supports Ariel.
+        justification : string
+            left, right or center.
+        bold : bool
+        italic : bool
+        shadow : bool
+
+        Returns
+        -------
+        text_actor : actor2d
+
+        """
+        text_actor = TextActor2D()
+        text_actor.set_position(position)
+        text_actor.message(text)
+        text_actor.font_size(font_size)
+        text_actor.font_family(font_family)
+        text_actor.justification(justification)
+        text_actor.font_style(bold, italic, shadow)
+        if vtk.vtkVersion.GetVTKSourceVersion().split(' ')[-1] <= "6.2.0":
```

Comment:
This is a really specific version number. Just making sure, those properties were exactly introduced in VTK 6.3.0 ?